src/content/tools/pub/dependencies.md

```diff
@@ -151,6 +151,12 @@ dependencies:
     git: git@github.com:munificent/kittens.git
 ```
 
+Alternatively, you can access private repos using access tokens via HTTPS.
+This bypasses the need to continously generate SSH keys in certain cases.
```

Comment:
I don't know of any _sane_ setups where you need to continuously generate SSH keys.

That sounds like a corner case.